tests/Aspire.Hosting.Azure.Tests/AzureFunctionsTests.cs

```diff
@@ -271,17 +271,18 @@ public async Task AddAzureFunctionsProject_WorksWithAddAzureContainerAppsInfrast
 
         await ExecuteBeforeStartHooksAsync(app, default);
 
-        var projRoles = Assert.Single(model.Resources.OfType<AzureProvisioningResource>().Where(r => r.Name == $"funcapp-roles"));
```

Comment:
Should we validate that the funcapp-identity resource emits the right manifest/bicep here? Particularly around the principal ID being an output that's consumed as a parameter elsewhere?

Reply:
I don't think that is necessary everywhere. We validate that in other places - specifically in the AzureContainerAppsTests.

https://github.com/dotnet/aspire/blob/122579052d497681ba156bb23058a70434eb1dd7/tests/Aspire.Hosting.Azure.Tests/AzureContainerAppsTests.cs#L2427-L2464

Feels like overkill to spread that all over.

Note that this test does verify that the principalId comes from the identity resource.

https://github.com/dotnet/aspire/blob/122579052d497681ba156bb23058a70434eb1dd7/tests/Aspire.Hosting.Azure.Tests/AzureFunctionsTests.cs#L286